index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content= "width=device-width, initial-scale=1.0">
+        <meta content="text" charset="utf-8">
+        <link rel="stylesheet" href="styles.css">
+        <title>Kiplaacollo Portfolio website</title>
+
+    </head>
+    <body>
+        <header class="head">
+                <h3 class="logo">Welcome</h3>
+                <img src="images/Enabled.png" alt="menu icon">
```

Comment:
- [ ] Kindly I recommend you to use `nav` tag for  hamburger button.The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.